src/plugins/spec/spec.c

```diff
@@ -578,7 +589,354 @@ static int handleErrors (Key * parentKey, KeySet * ks, Key * key, Key * specKey,
 	return ret;
 }
 
-static void copyMeta (Key * key, Key * specKey)
+
+// Naive approach to replace wildcards in meta string with the actual values
```

Comment:
Please use doxygen comments here. A bit more explaination would be helpful. Is this the work-in-progress we discussed?